src/Mealz/MealBundle/Entity/Day.php

```diff
@@ -123,16 +123,17 @@ public function getMeals()
     /**
      * @param ArrayCollection $meals
      */
-    public function setMeals($meals)
+    public function setMeals($meals): void
     {
         $this->meals = $meals;
     }
 
     /**
      * add a Meal
+     *
      * @param Meal $meal
```

Comment:
@param is redundant

Reply:
Yes, and a lot of stuff in this class too, but I just wanted to please psalm with void first. The plan was not to refactor everything.